src/main/scala/com/mwz/sonar/scala/util/MoreIssues.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2018-2019  All sonar-scala contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.mwz.sonar.scala
+package util
+
+object MoreIssues {
+  def MoreIssues = ???
```

Comment:
![info](https://static.sonar-scala.com/img/severity-info.svg 'Severity: info') Method name not recommended ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.naming.MethodNames))